lib/src/states/camera_picker_state.dart

```diff
@@ -926,7 +926,12 @@ class CameraPickerState extends State<CameraPicker>
         viewType: CameraPickerViewType.image,
       );
       if (entity != null) {
-        Navigator.of(context).pop(entity);
+        if (pickerConfig.onPicked case final onPicked?) {
+          await controller.resumePreview();
+          onPicked(entity);
```

Comment:
`resumePreview` doesn't seem to be enough at this place.

Reply:
done updated to `onPickConfirmed`.

Reply:
any suggestion for this?

Reply:
There are more steps after the return, so maybe we can put the pop step together with the return.

Reply:
done putting return together with pop step.